docs/DeveloperGuide.adoc

```diff
@@ -345,6 +345,60 @@ image::UndoRedoActivityDiagram.png[width="200"]
 **Cons:** Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things. +
 // end::undoredo[]
 
+// tag::list[]
+=== List (Mainlist, Blacklist etc) Mechanism
+
+List feature is a useful feature to view persons of a certain category. In the context of this project, master list contains all the contacts of the debt collection agency. As the need to categorize people into the respective groups was necessary, we implemented a new way to obtain this.
```

Comment:
I think you should keep your explanation solely to just describing the mechanism itself. How useful the feature is not important in this case. Keep it concise and remove phrases that do not add value to what you are trying to explain